extensions.json

```
@@ -222,6 +222,11 @@
       "id": "ctf0.macros",
       "repository": "https://github.com/ctf0/macros"
     },
+    {
+      "id": "damonsk.vscode-wardley-maps",
+      "repository": "https://github.com/damonsk/vscode-wardley-maps.git",
+      "version": "1.0.7"
```

No need to pin a specific version if we just publish from the default branch (i.e. not from a specific release branch or tag).

```suggestion
      "repository": "https://github.com/damonsk/vscode-wardley-maps"
```